File: Java/백준/Bronze/32776. 가희와 4시간의 벽 2/가희와 4시간의 벽 2.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        int s = 0;
        for (int i =0; i<3; i++){
            s +=in.nextInt();
        }
        if(s<t && t>240){
            System.out.println("flight");
        }
        else{
            System.out.println("high speed rail");
        }
    }
}

File: C#/백준/Bronze/32776. 가희와 4시간의 벽 2/가희와 4시간의 벽 2.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            if(aa.Sum()<t && t>240){
                Console.WriteLine("flight");
            }
            else{
                Console.WriteLine("high speed rail");
            }
        }
    }
}